addOns/reports/reports.gradle.kts

```diff
@@ -76,6 +76,7 @@ spotless {
             fileTree(projectDir) {
                 include("src/**/*.html")
                 exclude("src/main/zapHomeFiles/reports/risk-confidence-html/report.html")
+                exclude("src/main/zapHomeFiles/reports/traditional-html*/report.html")
```

Comment:
Had to exclude it because it didn't like the seemingly unmatched `tr` tags in the summary table.